ttnn/cpp/ttnn/operations/eltwise/complex_unary_backward/complex_unary_backward_pybind.hpp

```diff
@@ -39,6 +39,8 @@ void bind_complex_unary_backward(py::module& module, const complex_unary_backwar
         Returns:
             List of ttnn.Tensor: the output tensor.
 
+        Note:
+            {3}
```

Comment:
You can remove memory config defaults to None. It is not mentioned in the arguments